TeamCode/src/main/java/org/firstinspires/ftc/teamcode/robot_components/robot/DuckSpinner.java

```diff
@@ -20,7 +20,6 @@
     private ElapsedTime runtime = null;
     private Optional<Telemetry> telemetry; // used to log power if not null
                                            // Optional<e> prevents NullReferenceExceptions
-
     private boolean isRunning = false;
```

Comment:
this code is ok for now, since it needs changing later for acceleration ramping.